Add support for U2F/FIDO

chrisburr opened the pull request:
Closes #13

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.


I do have some suggestions for making it better though...


For **recipe**:

 * License is not an SPDX identifier (or a custom LicenseRef) nor an SPDX license expression.

Documentation on acceptable licenses can be found [here]( https://conda-forge.org/docs/maintainer/adding_pkgs.html#spdx-identifiers-and-expressions ).

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

chrisburr commented:
@conda-forge/openssh Ready for review

This PR adds support for U2F/FIDO (i.e. YubiKeys and other security tokens). I've also fixed the lint warning and added myself to the maintainers.

The only downside of this is that I've had to drop support for CentOS 6-like systems and build against the CentOS 7 based sysroot but hopefully that isn't an issue given it's been EOL for almost a year.
